### ISSUE TITLE
Fine-Tuning on HR images

### ISSUE THREAD

**Issue by alikashlan10** [ESAOpenSR/SRGAN#103]
it's a question rather than an issues

I'm trying to fine-tune an SR model on a high resolution imagery (0.5m , RGB-NIR , 16-bit) for a while and I don't get good results . the main problem that I have data from multiple sensors in DN values and I think this is what causes the model not to generalize well , my next move is to put all my training data in 8-bit to narrow down the differences between sensors and get out with a generalized model. and my main goal is the visual enhancement not the radiometric accuracy so what do you think about that ?

another question about the data preparation  , I have like 20 images (large dimension), I will tile them to produce a reasonable number of sample images enough for training and testing , should I normalize before or after the tiling ? and is this even the right pipeline to go with ?
Large HR image -> Normalize -> tile to small HR images -> generate small LR images 

and about the band-ordering , should all the images has the order RGB-NIR as band1:R , band2:G , band3:B , aband4:NIR ?

it will be very helpful from you to guide me through this and thanks in advance!
